metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPluginMetadata.java

```diff
@@ -0,0 +1,24 @@
+package eu.europeana.metis.core.workflow.plugins;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Two unused imports... :)

Reply:
Something has seriously gone wrong with my ide, the new update I guess, darn...